### PR TITLE
allow deployment of DS13 through github

### PR DESCRIPTION
**Pull request by netwmr01**
allow deployment of DS13 through github

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@netwmr01__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,5 @@ before_script:
   - npm install -g mocha
 node_js:
   - "0.12"
+sudo: required
```
Comment (singhkay) [inline .travis.yml]: @sedouard Can you review this update?

**Comment by singhkay** [comment]
Confirmed with @kmouss that this works offline. Merging.

File: .travis.yml (hunk 1 above)
Comment (netwmr01) [inline .travis.yml]: this sudo required can be removed. I was trying to allow sudo to see if this was causing CI failure.

**Pull request merged**